src/acceptance-test/java/org/zalando/nakadi/webservice/TimelinesControllerAT.java

```diff
@@ -6,28 +6,36 @@
 import com.jayway.restassured.http.ContentType;
 import org.apache.http.HttpStatus;
 import org.hamcrest.Matchers;
+import org.joda.time.DateTime;
 import org.json.JSONObject;
-import org.junit.AfterClass;
-import org.junit.BeforeClass;
 import org.junit.Test;
 import org.zalando.nakadi.domain.EventType;
+import org.zalando.nakadi.domain.EventTypeOptions;
 import org.zalando.nakadi.utils.EventTypeTestBuilder;
 import org.zalando.nakadi.webservice.utils.NakadiTestUtils;
 
+import java.time.Duration;
+
+import static java.time.temporal.ChronoUnit.DAYS;
+import static java.time.temporal.ChronoUnit.MINUTES;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.zalando.nakadi.util.DateWithinMatcher.dateWithin;
+
 public class TimelinesControllerAT extends BaseAT {
 
-    private static EventType eventType = EventTypeTestBuilder.builder().build();
+    private static final long RETENTION_TIME_MS = Duration.of(2, DAYS).toMillis();
 
-    @BeforeClass
-    public static void setUp() throws JsonProcessingException {
-        NakadiTestUtils.createEventTypeInNakadi(eventType);
-    }
+    private final EventType eventType;
 
-    @AfterClass
-    public static void tearDown() {
```

Comment:
@adyach In our acceptance tests we almost never clean (except user journey).

My opinion is that cleaning-after + tests-that-assume-the-DB-is-empty is a bad approach. It will always lead to some tests that forget to clean and some other tests fail because of that.

I think the better approach is not to clean after tests + all tests assume that DB is dirty + the tests that need clean-DB clean it in `@Before`. This is actually the approach we have now in Nakadi; I think for us it works fine.